deploy/01-chaos-operator.yaml

```diff
@@ -316,9 +316,9 @@ spec:
           type: object
   version: v1alpha1
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
+    - name: v1alpha1
```

Comment:
Are the spaces added via linting tool or self added?

Reply:
Lint